src/lib.rs

```diff
@@ -40,16 +40,27 @@ pub fn parse(contents: &str) -> Result<EditorConfigINIAST, Error<Rule>> {
 							ast.body.push(Item::Section(current_section.unwrap()));
 						}
 						current_section = Some(Section {
-							name: line.into_inner().next().unwrap().as_str().to_string(),
+							header: SectionHeader {
+								name: line.into_inner().next().unwrap().as_str().to_string(),
+								raws: Raws {
+									before: Some("".to_string()),
```

Comment:
These are temporary until I can actually have confidence I'm using the right tokens. It compiles, but panics at runtime at the moment, because I'm unwrapping the wrong tokens (new whitespace tokens).

Reply:
:thinking: I haven't run it yet.